.github/workflows/test.yaml

```diff
@@ -69,6 +69,7 @@ jobs:
           sudo apt clean
           docker rmi $(docker image ls -aq)
           df -h
+          docker volume prune -f
```

Comment:
nit: could you move it above the `df -h` line so we see the total disk space available after we also pruned the volumes?

Reply:
Changed the command to `docker system prune -af` and removed `docker rmi $(docker image ls -aq)` - system prune removes all unused data.

Reply:
According to https://docs.docker.com/engine/reference/commandline/system_prune/, we need to pass `--volumes` to the prune command to include volumes.

Reply:
Done. Thanks 💙